### PR TITLE
WhatsApp credit ratios doc (hidden)

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_hidden/other/credit_ratios.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+| Rest of Middle East | 19.5 | 6 |
+| Rest of Western Europe | 35 | 10.5 |
+| Russia | 17.5 | 10.5 |
+| Saudi Arabia | 8.5 | 5 |
+| South Africa | 7.5 | 4.5 |
+| Spain | 16.5 | 10 |
+| Turkey | 2.5 | 1 |
+| United Arab Emirates | 8.5 | 5 |
+| United Kingdom | 17| 10.5 |
+
```
Comment (KellieHawks) [inline _docs/_hidden/other/credit_ratios.md:51]:
```suggestion
{: .reset-td-br-1 .reset-td-br-2}

```

File: _docs/_hidden/other/credit_ratios.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Algeria, Angola, Benin, Botswana, Burkina Faso, Burundi, Cameroon, Chad, Congo, Eritrea, Ethiopia, Gabon, Gambia, Ghana, Guinea-Bissau, Ivory Coast, Kenya, Lesotho, Liberia, Libya, Madagascar, Malawi, Mali, Mauritania, Morocco, Mozambique, Namibia, Niger, Rwanda, Senegal, Sierra Leone, Somalia, South Sudan, Sudan, Swaziland, Tanzania, Togo, Tunisia, Uganda, Zambia
+
+#### Rest of Asia Pacific
+Afghanistan, Australia, Bangladesh, Cambodia, China, Hong Kong, Japan, Laos, Mongolia, Nepal, New Zealand, Papua New Guinea, Philippines, Singapore, Sri Lanka, Taiwan, Tajikistan, Thailand, Turkmenistan, Uzbekistan, Vietnam
+
+#### Rest of Central & Eastern Europe
+Albania, Armenia, Azerbaijan, Belarus, Bulgaria, Croatia, Czech Republic, Georgia, Greece, Hungary, Latvia, Lithuania, Macedonia, Moldova, Poland, Romania, Serbia, Slovakia, Slovenia, Ukraine
+
+#### Rest of Latin America
+Bolivia, Costa Rica, Dominican Republic, Ecuador, El Salvador, Guatemala, Haiti, Honduras, Jamaica, Nicaragua, Panama, Paraguay, Puerto Rico, Uruguay, Venezuela.
```
Comment (lydia-xie) [inline _docs/_hidden/other/credit_ratios.md:68]:
```suggestion
Bolivia, Costa Rica, Dominican Republic, Ecuador, El Salvador, Guatemala, Haiti, Honduras, Jamaica, Nicaragua, Panama, Paraguay, Puerto Rico, Uruguay, Venezuela
```

File: _docs/_hidden/other/credit_ratios.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+| Italy | 17| 10 |
+| Malaysia | 19.5 | 6 |
+| Mexico | 9.5 | 3 |
+| Netherlands | 39.5 | 23.5 |
+| Nigeria | 13.5 | 8 |
+| North America | 4| 2.5 |
+| Other | 13 | 4 |
+| Pakistan | 12.5 | 4 |
+| Peru | 16 | 4.5 |
+| Rest of AFRICA | 32 | 9.5 |
```
Comment (lydia-xie) [inline _docs/_hidden/other/credit_ratios.md:38]:
```suggestion
| Rest of Africa | 32 | 9.5 |
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
lgtm! just caught 2 minor things for your review

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**